pkg/env/dev.go

```diff
@@ -158,7 +158,7 @@ func (d *dev) InitializeAuthorizers() error {
 }
 
 func (d *dev) ACRName() string {
-	return "arosvc"
+	return "arointsvc"
```

Comment:
check that the arointsvc ACR is replicated to all of the regions where we have development environments.

Reply:
good point. it is currently sync'd to centralus and eastus only

Reply:
https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/7293992